.github/actions/spelling/line_forbidden.patterns

```diff
@@ -30,19 +41,22 @@
 \b[Nn]o[nt][- ]existent\b
 
 # s.b. preexisting
-[Pp]re-existing
+[Pp]re[- ]existing
 
 # s.b. preempt
-[Pp]re-empt\b
+[Pp]re[- ]empt\b
 
 # s.b. preemptively
-[Pp]re-emptively
+[Pp]re[- ]emptively
 
 # s.b. reentrancy
-[Rr]e-entrancy
+[Rr]e[- ]entrancy
 
 # s.b. reentrant
-[Rr]e-entrant
+[Rr]e[- ]entrant
+
+# s.b. workaround(s)
+#\bwork[- ]arounds?\b
```

Comment:
Similarly, I'm leaving this out. It's right some of the time (including once below).